react-search-component/my-app/src/List.tsx

```diff
@@ -0,0 +1,51 @@
+import './List.css';
+
+type StringList = {
```

Comment:
I'd prefer to see the word Props in this type name to make it more clear what it is used for.